newdle/client/src/components/NewdlesParticipating.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import flask from 'flask-urls.macro';
+import {Container, Icon, Label, Placeholder} from 'semantic-ui-react';
+import {useHistory} from 'react-router';
+import {serializeDate, toMoment} from '../util/date';
+import client from '../client';
+import {usePageTitle} from '../util/hooks';
+import styles from './MyNewdles.module.scss';
+
+export default function NewdlesParticipating() {
+  const [participations, loading] = client.useBackend(() => client.getNewdlesParticipating(), []);
+  usePageTitle("Newdles I'm participating in");
+
+  let content;
+  if (loading || participations === null) {
+    content = (
+      <>
+        <Placeholder className={styles.newdle} />
+        <Placeholder className={styles.newdle} />
+        <Placeholder className={styles.newdle} />
+      </>
+    );
+  } else if (participations.length === 0) {
+    content = (
+      <div className={styles['no-newdle-container']}>
+        {/* eslint-disable-next-line jsx-a11y/accessible-emoji */}
```

Comment:
not needed, there aren't any emoji in the next line

Reply:
Not sure how that got through. Thanks!